de.metas.handlingunits.base/src/main/java/de/metas/picking/service/impl/HU2PackingItemsAllocator.java

```diff
@@ -519,7 +519,7 @@ private void pickFromVHU(@NonNull final I_M_HU pickFromVHU, @NonNull final Packi
 		final StockQtyAndUOMQty stockQtyAndUomQty = CatchWeightHelper.extractQtys(_huContext, getProductId(), qtyPacked, pickFromVHU);
 
 		// "Back" allocate the qtyPicked from VHU to given shipment schedule
-		huShipmentScheduleBL.addQtyPickedAndUpdateHU(shipmentSchedule, stockQtyAndUomQty, pickFromVHU, _huContext);
+		huShipmentScheduleBL.addQtyPickedAndUpdateHU(shipmentSchedule, stockQtyAndUomQty, pickFromVHU, _huContext, false);
```

Comment:
follow-up:
```suggestion
		huShipmentScheduleBL.addQtyPickedAndUpdateHU(shipmentSchedule, stockQtyAndUomQty, pickFromVHU, _huContext, false/*IsAnonyouslyPickedOnTheFly*/);
```
not everyone has intelliJ displaying the parameter names nicely :-)